lib/oli/publishing/authoring_resolver.ex

```diff
@@ -362,4 +356,88 @@ defmodule Oli.Publishing.AuthoringResolver do
       )
     )
   end
+
+  @fragment """
+            SELECT jsonb_agg(
+              jsonb_build_object(
+                'idref', (elem->>'idref')::INT,
+                'href', split_part(elem->>'href', '/', 4)
+              )
+            )
+            FROM LATERAL jsonb_path_query(
+              ?, 'strict $.** \\? ((@.type == "a" && @.linkType == "page") || @.type == "page_link")'
+            ) AS elem
+            """
+            |> String.replace(~r/\s+/, " ")
```

Comment:
This line is meant to help the logger display the query in one line. It doesn't add any business logic.